sematic/plugins/abstract_builder.py

```diff
@@ -0,0 +1,58 @@
+"""
+The module that contains the definition of the AbstractBuilder plugin, which packages user
+code and builds container images that can be used to execute the user's funcs.
+"""
+# Standard Library
+import abc
+from typing import Type, cast
+
+# Sematic
+from sematic.abstract_plugin import AbstractPlugin, PluginScope
+from sematic.config.settings import get_active_plugins
+
+
+class BuildError(Exception):
```

Comment:
Consider moving to `exceptions.py`?

Reply:
Initially that's where I defined this, but then I thought that this would not need to be imported anywhere else except for the CLI, which already imports the builder.

If there any other reason why this should be moved there?

Reply:
No, other than standardization. It's a nit, do what you like.